### ISSUE TITLE
error when starting the application for first time ?

### ISSUE THREAD

**Issue by nahush2321** [kmatheussen/radium#1371]
hello i have small problem as when when  startradium i get message that it can not start as /opt/radium/ladspa files are not present and please check if all files from zip is unpacked. i have attached my terminal output. Also thi is self compiled version on arch linux and was first time runnin application on this machine also not ever tried any of the previous versions .

[radium.txt](https://github.com/kmatheussen/radium/files/7986103/radium.txt)

**Comment by akimaze** [comment]
I think that files are only in official binary package.

**Comment by kmatheussen** [comment]
It should work just making an empty directory called "ladspa" in the "bin" directory before running "./install.sh". Did you use the official arch build or did you build it yourself? I'm surprised if the official arch build didn't work. I actually thought "./install" should always work.

**Comment by kmatheussen** [comment]
I guess install.sh doesn't fix this. Just append these lines to "install.sh":

```
# ladspa
if [ ! -d "$TARGET/ladspa" ]
then
    mkdir $TARGET/ladspa
fi
```

**Comment by kmatheussen** [comment]
> I think those files are only in official binary package.

Yes, the official binaries include ladspa plugins from this package:
https://github.com/kmatheussen/common-ladspa-plugins
